contracts/utils/timelock/QuadraticTokenTimelock.sol

```diff
@@ -8,14 +8,21 @@ contract QuadraticTokenTimelock is TokenTimelock {
     constructor (
         address _beneficiary,
         uint256 _duration,
-        address _lockedToken
+        address _lockedToken,
+        uint256 _cliffDuration,
+        address _clawbackAdmin,
+        uint256 _startTime
     ) TokenTimelock(
         _beneficiary, 
         _duration, 
-        7776000, // 3 months cliff
+        _cliffDuration,
         _lockedToken, 
-        address(0xB8f482539F2d3Ae2C9ea6076894df36D1f632775) // fei labs multisig
-    ) {}
+        _clawbackAdmin
+    ) {
+        if (_startTime != 0) {
+            startTime = _startTime;
+        }
```

Comment:
it should revert like crazy if we set a startTime in the future, because of checks like this in `Timed.sol`  ?
```
uint256 timePassed = block.timestamp - startTime; // block timestamp always >= startTime
```

Reply:
This is fine imo. The use case here is actually in the *past* for rari founders who started vesting on October 2020